app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/MainNavGraph.kt

```diff
@@ -0,0 +1,79 @@
+package com.bottlerocketstudios.brarchitecture.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavOptionsBuilder
+import androidx.navigation.compose.composable
+import androidx.navigation.navigation
+import com.bottlerocketstudios.brarchitecture.ui.auth.AuthCodeViewModel
+import com.bottlerocketstudios.brarchitecture.ui.auth.toState
+import com.bottlerocketstudios.brarchitecture.ui.devoptions.DevOptionsViewModel
+import com.bottlerocketstudios.brarchitecture.ui.devoptions.toState
+import com.bottlerocketstudios.brarchitecture.ui.home.HomeViewModel
+import com.bottlerocketstudios.brarchitecture.ui.home.toState
+import com.bottlerocketstudios.brarchitecture.ui.splash.SplashViewModel
+import com.bottlerocketstudios.compose.auth.AuthCodeScreen
+import com.bottlerocketstudios.compose.devoptions.DevOptionsScreen
+import com.bottlerocketstudios.compose.home.HomeScreen
+import com.bottlerocketstudios.compose.splash.SplashScreen
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.FlowCollector
+import org.koin.androidx.viewmodel.ext.android.getViewModel
+
+// TODO - Create utils file
```

Comment:
Agree with this todo (prior to finalization/completion of using compose navigation everywhere) :)